### PR TITLE
DE6343: We Need That Content Block:tm:

### PR DESCRIPTION
**Pull request by lukemoderwell**
guess we lost the content block. This should get it back.

### CODE REVIEW

File: src/app/services/contentful.service.ts [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
 
 export class ContentfulService {
   private client;
   constructor() {
-    this.client = createClient({
-      space: environment.CONTENTFUL_SPACE_ID,
-      accessToken: environment.CONTENTFUL_ACCESS_TOKEN,
-      environment: environment.CONTENTFUL_ENV
-    });;
+    this.client = createClient({space: environment.CONTENTFUL_SPACE_ID, accessToken: environment.CONTENTFUL_ACCESS_TOKEN, environment: environment.CONTENTFUL_ENV});;
```
Comment (j-brammer) [inline src/app/services/contentful.service.ts]: Is there a reason you flattened out the code here and on line 8? Not my favorite as far as formatting goes.

File: src/app/services/contentful.service.ts [hunk 2]
```diff
@@ -1,29 +1,27 @@
...
   }
 
-  public getSuggestedContentBlock(id): Observable<Suggested> {
-    return from(this.client.getEntry(id)).pipe(
-     map((entry: any) => {
-       return new Suggested(entry.fields.content);
-     }) 
-    )
+  public getSuggestedContentBlock() : Observable < Suggested > {
+    return from(this.client.getEntries({ "sys.contentType.sys.id" : "content_block", "limit": 1000})).pipe(map((data : any) => {
```
Comment (j-brammer) [inline src/app/services/contentful.service.ts]: What is the key we are searching for for the "suggested" content block? Seems really excessive to loop through eveyrthing until you find it versus just asking Contentful directly for the hard coded content block.

File: src/app/services/contentful.service.ts (hunk 1 above)
Comment (lukemoderwell) [reply src/app/services/contentful.service.ts]: that's what my formatter did. Sometimes it does good stuff and sometimes bad.

Comment (j-brammer) [reply src/app/services/contentful.service.ts]: 👍

File: src/app/services/contentful.service.ts (hunk 2 above)
Comment (j-brammer) [reply src/app/services/contentful.service.ts]: After conversation this is a limitation of how we can ask Contentful for information

**Pull request merged**